src/FormMigration/Steps/FundsAndDesignations.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class FundsAndDesignations extends FormMigrationStep
+{
+    /**
+     * @unreleased
+     */
+    public function canHandle(): bool
+    {
+        $fundsAndDesignationsAttributes = $this->getFundsAndDesignationsAttributes($this->formV2->id);
```

Comment:
Another option for the migration step is to add update the `FormDecorator` so you could do something this:

```
return $this->formV2->hasFunds() || $this->formV2->hasFundOptions()
```

Although what you have works, it's a little hard to understand the logic when reading it - so updating would mainly be for comprehension.

Reply:
Resolve in 78be4079f1976a168446de7d5364219f2f655c7a